projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -688,13 +937,104 @@ export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation
+    },
+    {
+      class: 'CommandService',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'create',
+      comment:
+        "The 'create' method's signature changed to: 'create( commandFactory: (command: PARAMS) => Observable<any> options: { strategy?: CommandStrategy; }): Command<PARAMS, RESULT>'",
+    },
     {
       class: 'DeleteUserAddressEvent',
       importPath: '@spartacus/core',
       deprecatedNode: 'userId',
       comment:
         "// TODO:Spartacus - Property 'userId' was removed from Class 'DeleteUserAddressEvent'. ",
     },
+    {
+      class: 'GlobalMessageConfig',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'globalMessages',
+      comment:
+        "The type of property 'globalMessages' changed to: 'globalMessages: {\n        [GlobalMessageType.MSG_TYPE_CONFIRMATION]?: GlobalMessageTypeConfig;\n        [GlobalMessageType.MSG_TYPE_INFO]?: GlobalMessageTypeConfig;\n        [GlobalMessageType.MSG_TYPE_ERROR]?: GlobalMessageTypeConfig;\n        [GlobalMessageType.MSG_TYPE_WARNING]?: GlobalMessageTypeConfig;\n        [GlobalMessageType.MSG_TYPE_ASSISTIVE]?: GlobalMessageTypeConfig;\n    }' ",
+    },
+    {
+      class: 'MultiCartService',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'createCart',
+      comment:
+        "The 'createCart' method's signature changed to: 'createCart( { userId, oldCartId, toMergeCartGuid, extraData, }: { userId: string; oldCartId?: string; toMergeCartGuid?: string; extraData?: { active?: boolean; }; }): Observable<Cart>'",
+    },
+    {
+      class: 'MultiCartService',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'getCartEntity',
+      comment:
+        "The 'getCartEntity' method's signature changed to: 'getCartEntity( cartId: string): Observable<StateUtils.ProcessesLoaderState<Cart | undefined>>'",
+    },
+    {
+      class: 'MultiCartService',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'getEntry',
+      comment:
+        "The 'getEntry' method's signature changed to: 'getEntry( cartId: string productCode: string): Observable<OrderEntry | undefined>'",
+    },
+    {
+      class: 'MultiCartService',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'getLastEntry',
+      comment:
+        "The 'getLastEntry' method's signature changed to: 'getLastEntry( cartId: string productCode: string): Observable<OrderEntry | undefined>'",
+    },
+    {
+      class: 'OccCartAdapter',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'load',
+      comment:
+        "The 'load' method's signature changed to: 'load( userId: string cartId: string): Observable<Cart | undefined>'",
+    },
+    {
+      class: 'OccCartVoucherAdapter',
+      importPath: '@spartacus/core',
+      deprecatedNode: 'getCartVoucherEndpoint',
+      comment:
+        "The 'getCartVoucherEndpoint' method's signature changed to: 'getCartVoucherEndpoint( userId: string cartId: string): string'",
```

Comment:
not a problem, but to mention that this is a case of a method's param not having a property before, therefore was implicitly typed as any for the `cartId`, but now it is explicitly typed as a string.  --> but again we can keep it and not really a breaking change

Reply:
Correct.  Let's discuss if it's ok to leave those cases in.  We could document the fact that some changes in the list are "potential" breaking changes depending on how the code is used and extended.